tests/integration/pilot/cni/cni_test.go

```diff
@@ -65,6 +65,10 @@ func TestCNIReachability(t *testing.T) {
 					ConfigFile: "global-mtls-on.yaml",
 					Namespace:  systemNM,
 					Include: func(src echo.Instance, opts echo.CallOptions) bool {
+						// Exclude headless naked service, because it is no sidecar
```

Comment:
Why is this skipped rather than just expectSuccess=false?

Reply:
Yeah, we can. It just ignores it as some other cases are in security but not here